src/main/resources/embedding-providers-config.yaml

```diff
@@ -10,6 +10,16 @@ jsonapi:
         supported-authentication:
           - "HEADER"
           - "SHARED_SECRETS"
+        models:
+          - name: text-embedding-3-small
+            properties:
+              vector-dimension: 1536
+          - name: text-embedding-3-large
+            properties:
+              vector-dimension: 3072
+          - name: text-embedding-ada-002
+            properties:
+              vector-dimension: 1536
 
```

Comment:
We previously defined `properties` as internal use to store timeout, retry etc. If we store `vector-dimension` under the `properties`, users cannot get the vector dimension if they call `findEmbeddingServices`. Will it be better to make `vector-dimension` to the same level as `name` and `properties`?

Reply:
It would be very convenient to be able to see `vector-dimension` in `findEmbeddingServices`, so I think this would be reasonable